services/ansible/ansible-collection-version.service.js

```diff
@@ -0,0 +1,51 @@
+import Joi from 'joi'
+import { renderVersionBadge } from '../version.js'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const ansibleCollectionSchema = Joi.object({
+  deprecated: Joi.boolean().required(),
+  highest_version: Joi.object({
+    version: Joi.string().required(),
+  }).required(),
+  // Ansible docs don't mention this but it appears in the API responses
+  download_count: Joi.number().required(),
+}).required()
```

Comment:
We should only validate the values we actually need for the badge. So here we should only validate
`highest_version.version` and in the other one, we should only validate `download_count`.
If we got an API response with no `deprecated` key in it but the values we need are there, we don't need to throw an error.

Reply:
Thanks, yeah I originally had it in 1 file to build it out then refactored. Missed adjusting this back